### DISCUSSION TITLE
Delete hotkey

### DISCUSSION THREAD

**Discussion by AlanEdd** [Flow-Launcher/Flow.Launcher#2867] [Q&A]
Just wondering is there any delete hotkey, to delete a file or folder in context in the search results ? 
It would be handy to have `Shift+Delete` rather than, going to the context menu and so on. 

Thanks 
Alan.

**Comment by jjw24** [comment]
I don't believe so. This will be w good enhancement for Explorer plugin.

**Comment by newnol** [comment]
On the 1.19.2 version, i see it have a option menu when you right click to the file, it has the option to delete the file
![image](https://github.com/user-attachments/assets/ddb18766-770b-42b0-8c87-0b68ac429a3a)

**Reply by jjw24** [reply]
Yes there is the function provided by the context menu, but there isn't a keyboard hotkey to press for deletion.

**Comment by taooceros** [comment]
The major problem is we don't have a per-result hotkey hook yet. This will take some major effort to make.
